tests/platform_tests/daemon/test_chassisd.py

```diff
@@ -77,13 +77,14 @@ def check_if_daemon_restarted(duthost, daemon_name, pre_daemon_pid):
 
 def collect_data(duthost):
     keys = duthost.shell('sonic-db-cli STATE_DB KEYS "CHASSIS_*TABLE|*"')['stdout_lines']
+    sorted_keys = sorted(keys)
 
     dev_data = {}
-    for k in keys:
+    for k in sorted_keys:
         data = duthost.shell('sonic-db-cli STATE_DB HGETALL "{}"'.format(k))['stdout']
         data = compose_dict_from_cli(data)
         dev_data[k] = data
-    return {'keys': keys, 'data': dev_data}
+    return {'keys': sorted_keys, 'data': dev_data}
```

Comment:
hi @mannytaheriif this collect_data is called by wait_data to get data_after_restart, but it's initialized as unordered dictionary, so order would lose, please help to review https://github.com/sonic-net/sonic-mgmt/pull/9897